KCs-Linux-Scripts/Linux-Auto-Compile-Best-Of-Both-Worlds.sh

```diff
@@ -0,0 +1,49 @@
+#/bin/bash
+
+cd ~
+
+git clone https://github.com/wabbajack-tools/ussedp
+
+cd ussedp
+
+if [ ! -f "ussedp-patches.7z" ]
+then
+    wget --load-cookies /tmp/cookies.txt "https://docs.google.com/uc?export=download&confirm=$(wget --quiet --save-cookies /tmp/cookies.txt --keep-session-cookies --no-check-certificate 'https://docs.google.com/uc?export=download&id=1lDxHD_bDwltsp88fhazXTngVozr6TDXh' -O- | sed -rn 's/.*confirm=([0-9A-Za-z_]+).*/\1\n/p')&id=1lDxHD_bDwltsp88fhazXTngVozr6TDXh" -O ussedp-patches.7z && rm -rf /tmp/cookies.txt
+else
+    echo "patch files already downloaded"
+fi
+
+if [ ! -d "/home/$(whoami)/ussedp/BestOfBothWorlds" ]
+then
+    7z x -y 'ussedp-patches.7z'
+else
+    echo "patch files already installed"
+fi
+
+echo "Enter Your sudo Password Below:"
+
+sudo dotnet build -c release
+
+sudo rsync -avx ~/ussedp/BestOfBothWorlds/ ~/ussedp/Patcher/bin/Release/net6.0/
+
+if [ -d "/home/$(whoami)/Linux-ussedp-BestOfBothWorlds" ]
+then
+    rm -r ~/Linux-ussedp-BestOfBothWorlds/
+else
+    echo "dir ready"
+fi
+
+rsync -avx ~/ussedp/Patcher/bin/Release/net6.0/ ~/Linux-ussedp-BestOfBothWorlds/
```

Comment:
doesn't this need sudo too?

Reply:
and what about full patcher

Reply:
no actually `rsync` will copy anything is can read so we only need `sudo` for sending the files

Reply:
If you want test and resolve this review on your distro, that would be better so we can be sure that it doesn't actually need it, I know `rsync` works like that on Manjaro and kubuntu so...

Reply:
> and what about full patcher

It's up now I just needed some time to test it

Reply:
worked

Reply:
cool